CHANGELOG.md

```diff
@@ -53,13 +53,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Changed GraphQL API `preserveOutliers` to use `PreserveOutliersOutput` in its response.
   - Instead of returning the count of successfully marked outliers, this endpoint
     now returns a list of outliers that were not marked as saved.
-- In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, `TidbRule`,
-  `BlockListLdap`,`LdapPlainText`,`LdapBruteForce`,`BlockListKerberos`,
-  `HttpThreat`,`TorConnection`,`DomainGenerationAlgorithm`,`NonBrowser`,
-  `BlockListHttp`,`ExtraThreat`, `BlockListConn`, `BlockListDceRpc`,
-  `DnsCovertChannel`, `LockyRansomware`,`CryptocurrencyMiningPool`,
-  `BlockListDns`, `FtpPlainText` and `BlockListFtp` changed GraphQL APIs to
-  return `StringNumber` or `ID` instead of integers beyond `i32`.
+- Changed GraphQL APIs to return `StringNumber` or `ID` instead of integers
+  beyond `i32`in all applicable APIs.
```

Comment:
There should be a space between `i32` and in.

Reply:
Done